wazuh/config/run.sh

```diff
@@ -29,6 +34,9 @@ exec_cmd() {
     eval $1 > /dev/null 2>&1 || error_and_exit "$1"
 }
 
+exec_cmd_stdout() {
```

Comment:
I added that because it's very useful to see which file was copied where by the `exec_cmd_stdout "cp --verbose -r $WAZUH_CONFIG_MOUNT/* $DATA_PATH"` command!